README.md

```diff
@@ -374,9 +373,9 @@ var pjax = new Pjax({
   switchesOptions: {
     ".js-Pjax": {
       classNames: {
-        // class to be removed from the element during replacement
+        // class added to the old element being replaced, ie: a fade out
```

Comment:
`i.e.` should be `e.g.`, and the same below.

`i.e.` = in other words.
`e.g.` = an example.

Reply:
@BehindTheMath will do!

Reply:
@BehindTheMath fixed :)